package.json

```diff
@@ -29,6 +29,7 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "module": "lib/esm/index.js",
+  "type": "commonjs",
```

Comment:
This doesn’t actually do anything as is, but having a placeholder makes using `sed` a lot easier.